src/Signer/Ecdsa/PointsManipulator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Signer\Ecdsa;
+
+/**
+ * @internal
+ */
+interface PointsManipulator
```

Comment:
Can you please still describe the interface? Unclear what it does, or what the naming means. If it comes from somewhere, `@link` is vital.